app/views/render_async/_render_async.html.erb

```diff
@@ -4,10 +4,21 @@
 
 <% content_for :render_async do %>
   <%= javascript_tag html_options do %>
-    (function($){
-      $.ajax({ url: "<%= path %>" }).always(function(response) {
-        $("#<%= container_id %>").replaceWith(response);
-      });
-    }(jQuery));
+    var request = new XMLHttpRequest();
```

Comment:
I think it would make sense to wrap this in a function to keep the state out of global scope